components/ActionSheet/src/private/MDCActionSheetItemTableViewCell.m

```diff
@@ -185,10 +185,8 @@ - (void)setActionTextColor:(UIColor *)actionTextColor {
 }
 
 - (void)setRippleColor:(UIColor *)rippleColor {
-  _rippleColor = rippleColor;
-
-  // If no ripple color then reset to the default ripple color.
-  self.rippleTouchController.rippleView.rippleColor = rippleColor ?: RippleColor();
+  _rippleColor = rippleColor ?: RippleColor();
+  self.rippleTouchController.rippleView.rippleColor = rippleColor;
```

Comment:
With this change couldn't a client pass in `nil` and then we would be passing `nil` to `rippleView.rippleColor` which is `nonnull`?

Reply:
Good catch, fixed!